diff --git a/audisp/Makefile.am b/audisp/Makefile.am
index 2cedd54..877cb7f 100644
--- a/audisp/Makefile.am
+++ b/audisp/Makefile.am
@@ -21,7 +21,7 @@
 #   Steve Grubb <sgrubb@redhat.com>
 #
 
-SUBDIRS = plugins 
+SUBDIRS = 
 CONFIG_CLEAN_FILES = *.rej *.orig
 AM_CPPFLAGS = -D_GNU_SOURCE -fPIC -DPIC -I${top_srcdir} -I${top_srcdir}/lib -I${top_srcdir}/src -I${top_srcdir}/src/libev
 LIBS = ${top_builddir}/lib/libaudit.la
